x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -108,6 +125,11 @@ interface DraggableProps extends BaseProps {
    * be used if the element will be dropped into a text field.
    */
   label: string;
+  /**
+   * The event handler that fires when the user navigates to the next group
+   * with the current draggable picked up
+   */
+  onNextGroup?: () => GroupMoveResult;
```

Comment:
I think you already know this, but commenting to clarify. I think the concept of "next" is not something that can be an argument to the DragDrop component. If the user starts dragging in the field list, the "next" group should be defined as the workspace, and then the first group in the dimension list. This crosses too many boundaries to be passed in as a property.